build-vm.sh

```diff
@@ -111,36 +111,65 @@ fi
 # Check requirements
 reqsok=true
 
-haspacker=$(which packer)
-if [ -z "$haspacker" ]; then
+hash mvn &>/dev/null
```

Comment:
I disagree.  `hash` is a POSIX standard, shell built-in, and is widely used for this purpose.

Reply:
I used to agree on `hash` too but it appears that it has some caveats and that `command` is more widely recommended as stable across POSIX implementations.
- it is known as not very well defined by POSIX and behaves sometimes differently
- its call affects the system
- it may not be able to answer correctly on some built-in commands, positively or falsely depending on the Shell implementation.

From the man page:
```The hash utility affects the way the current shell	environment remembers the locations of utilities found as described in Command Search and Execution . Depending on the arguments specified, it adds utility locations to its list of remembered locations or it purges the contents of	the list. When no arguments are specified, it reports on the contents of	the list.

Utilities provided as built-ins to the shell are not reported by hash.
```

Even if mainly using explicit BASH scripts, at least to avoid the dash, I'm trying to avoid using to much BASHism or breaking compliance with others (ZSH) for free.
For instance, with ZSH:
```$ hash hash
hash: no such command: hash
$ command -v hash                                                                                                                                                                                          hash
```

Reply:
changes applied in pull request #5